### ISSUE TITLE
Removed from F-Droid?

### ISSUE THREAD

**Issue by vairag22** [Sangwan5688/BlackHole#395]
This app used to be available on F-Droid but for some reason it has been removed. I'm not sure if devs are aware of it because README still says it's available there and I didn't find any issues on this repo mentioning this problem.

My initial assumption was that BlackHole was removed due to some DMCA order since "blackhole = piracy & piracy = bad", but other apps like Spotiflyer, Newpipe are still on F-Droid. So it doesn't make any sense.

Devs, please look into the matter. Thanks.

![Screenshot_2022-08-31-11-43-55-722-edit_us spotco fennec_dos](https://user-images.githubusercontent.com/71322586/187607266-a2b76225-7a89-4db5-8652-e417b5e994bd.jpg)

![Screenshot_2022-08-31-11-27-01-127-edit_us spotco fennec_dos](https://user-images.githubusercontent.com/71322586/187607068-3bd26822-908e-4d64-a0bc-684d35c00247.jpg)

**Comment by BrightDV** [comment]
Yes, because of #345. See on [F-Droid's Forum](https://forum.f-droid.org/t/what-happened-to-blackhole-a-music-app/18863/3)

**Comment by Sangwan5688** [comment]
Ya, I'm aware of the issue. It was caused by `device-info-plus` package present inside BlackHole, which gets your device arch info and uses it to update the app. The package was F-droid friendly earlier but in recent versions they have added some classes which aren't allowed on F-droid. They claim that they have removed those classes again, and I have updated that package. So BlackHole should come back again. If the issue still persists then I might have to remove the package completely and you will have to manually select your arch version while updating.
